src/ui/browserAction/popup.css

```diff
@@ -27,3 +27,13 @@ body {
 .needsSlotted > * {
   display: none;
 }
+
+.stack {
```

Comment:
I'm using a grid to stack things, so the size is inherited from the child elements, (in contrast to pos-absolute stacking). 

Also if we don't stack, we might show a blip with both at the same time on top of each other which is more noticable then when we stack them,.